src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/EqualsAvoidNullCheck.java

```diff
@@ -475,9 +475,10 @@ private static FieldFrame getObjectFrame(FieldFrame frame) {
      */
     private static boolean checkLineNo(DetailAST field, DetailAST objCalledOn) {
         boolean result = false;
+        final int minimumSymbolsBetween = 11;
```

Comment:
I need explanation on this magic number.
We need to leave comment there.

Reply:
done

Reply:
// We should specify columnNo passing condition in such a way, 
        // so that the minimal possible distance between field and objCalledOn
        // will be the maximal condition to pass this check.
        // The minimal distance between objCalledOn and field (of type String) initialisation
        // is calculated as follows: 
        // String(6) + space(1) + variableName(1) + assign(1) + 
        // anotherStringVatiableName(1) + semicolumn(1) = 11

@romani is it more clear explanation?

Reply:
"We should" ==> "Required for pitest coverage. We should ....."

anotherStringVatiableName ==> anotherStringVariableName

Please add:
`example: length of "String s=d;" is 11 symbols.